spyder/plugins/layout/container.py

```diff
@@ -196,11 +193,11 @@ def trigger(i=index, self=self):
                     layout_switch_action = self.get_action(name)
                 except KeyError:
                     layout_switch_action = self.create_action(
-                        name,
-                        text=name,
-                        triggered=trigger(),
-                        register_shortcut=False,
-                    )
+                            name,
+                            text=name,
+                            triggered=trigger(),
+                            register_shortcut=False,
+                        )

```

Comment:
```suggestion
                        name,
                        text=name,
                        triggered=trigger(),
                        register_shortcut=False,
                    )
```

The extra indentation doesn't seem necessary anymore.